README.md

```diff
@@ -1,2 +1,88 @@
 # business-analytics
 Tools to produce Riff Business analytic reports on RiffEdu and RiffPlatform deployments
+
+This repository uses a third-party package called [Metabase][] to visualize usage metrics for Mattermost. Two docker containers are run; one using the metabase image and another using the
+official mysql image. Commands are provided to restore sql database backups from Mattermost to the mysql container. 
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Database Backups
+
+Download sql backups from the Riff Learning backup location (ask for the URL if you don't have it), and place all .sql files in the folder called 'database-backups' in the root of this repo.
+
+### Prerequisites
+
+- [docker CE][docker-install] (Community Edition)
+- [docker-compose][docker-compose-install]
+- git
+- make
+
+
+## Running the Software
+The first step is to clone this repository. The following commands are defined in a Makefile, and can be run as follows:
+
+```
+make up
+```
+This command will create the mysql and metabase containers and show the logs for each
+
+```
+make restore
+```
+Once the `make up` command has completed (about 2-3 minutes), run this command. It will restore databases from all .sql files in the folder ./database-backups
```

Comment:
It's sort of hard to run `make restore` when `make up` is attached to the terminal, your instructions should therefore probably mention that you'll have to use another terminal to do that.

Reply:
Added that to the documentation.